control-plane/connect-inject/controllers/peering/peering_acceptor_controller.go

```diff
@@ -266,7 +266,7 @@ func (r *AcceptorController) updateStatus(ctx context.Context, acceptorObjKey ty
 			return err
 		}
 		if acceptor.Status.LatestPeeringVersion == nil || *acceptor.Status.LatestPeeringVersion < peeringVersion {
-			acceptor.Status.LatestPeeringVersion = pointer.Uint64(peeringVersion)
+			acceptor.Status.LatestPeeringVersion = ptr.To(uint64(peeringVersion))
```

Comment:
Similar question here - seems like `main` and https://github.com/hashicorp/consul-k8s/pull/4256 don't yet use `ptr`, do we need to sync those up?

Reply:
yup, just stuck on 1.3.x still 😅

Reply:
Thank you for this lovely yak shave 🙏🏻 🥲